src/rebar_prv_eunit.erl

```diff
@@ -73,11 +74,12 @@ do_tests(State) ->
             Error
     end.
 
-do_tests(State, Tests) ->
+run_tests(State, Tests) ->
+    T = translate_paths(State, Tests),
     EUnitOpts = resolve_eunit_opts(State),
-    ?DEBUG("eunit_tests ~p", [Tests]),
-    ?DEBUG("eunit_opts  ~p", [EUnitOpts]),
-    Result = eunit:test(Tests, EUnitOpts),
+    ?INFO("eunit_tests ~p", [T]),
+    ?INFO("eunit_opts  ~p", [EUnitOpts]),
```

Comment:
Is this intentionally left this way, or that was helping filter out the info you needed from the flood of `DEBUG` rebar3 generates?